source/_integrations/ollama_conversation.markdown

```diff
@@ -0,0 +1,40 @@
+---
+title: Ollama Conversation
+description: Instructions on how to integrate Ollama as a conversation agent
+ha_category:
+  - Voice
+ha_release: 2023.4
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@synesthesiam'
+ha_domain: ollama_conversation
+ha_integration_type: service
+---
+
+The Ollama integration adds a conversation agent powered by a local [Ollama](https://ollama.com/) in Home Assistant.
+
+This conversation agent is unable to control your house. The Ollama conversation agent can be used in automations, but not as a [sentence trigger](/docs/automation/trigger/#sentence-trigger). It can only query information that has been provided by Home Assistant. To be able to answer questions about your house, Home Assistant will need to provide Ollama with the details of your house, which include areas, devices and their states. 
```

Comment:
```suggestion
This conversation agent is unable to control your house. The Ollama conversation agent can be used in automations, but not as a [sentence trigger](/docs/automation/trigger/#sentence-trigger). It can only query information that has been provided by Home Assistant. To be able to answer questions about your house, Home Assistant will need to provide Ollama with the details of your house, which include areas, devices, and their states. 
```